src/test/java/com/google/sps/scheduler/SchedulerTest.java

```diff
@@ -80,14 +90,12 @@ public void justOneTask() {
     calendarEvents.add(eventA);
     calendarEvents.add(eventB);
 
-    List<Event> actualTasksEvents = schedule(calendarEvents, ZURICH_TIME_ZONE, day);
-    List<Event> expectedTasksEvents = Arrays.asList(
-        createEventWithSummary(
-            createDateTime(day, 13, 0, ZURICH_TIME_ZONE),
-            createDateTime(day, 13, 30, ZURICH_TIME_ZONE),
-            ZURICH_TIME_ZONE, TasksProvider.TASKS_SAMPLE_TITLES.get(0)));
+    List<Task> actualScheduledTasks = schedule(calendarEvents, sampleTasks, ZURICH_TIME_ZONE, day);
+    List<Task> expectedScheduledTasks = Arrays.asList(
+        createTaskWithDue(
+            createDateTime(day, 13, 0, ZURICH_TIME_ZONE)));
 
-    Assert.assertEquals(expectedTasksEvents, actualTasksEvents);
+    Assert.assertEquals(expectedScheduledTasks, actualScheduledTasks);
```

Comment:
Now that you don't have a test anymore that can assert the calendar events created, you should also add new tests a level higher up that can validate the events created and not just the tasks.

Reply:
Ok, I was in doubt whether to add them.